src/database/models/users.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (sequelize, DataTypes) => {
```

Comment:
When you are about to create new models and migrations, you can run:
`sequelize model:create --name User --attributes name:string,title:string,bio:string`
Notice `User` is not pluralized, Sequelize will do that for you in the database. This folder is named `users` instead of `user`

Reply:
Thanks. Changes effected